make.go

```diff
@@ -110,8 +130,8 @@ func makeDir(dir string) {
 	}
 }
 
-func makeOutputPath(dir, name string) string {
-	if runtime.GOOS == "windows" {
+func makeOutputPath(dir, name string, isWindowsOS bool) string {
+	if isWindowsOS {
```

Comment:
You have already set `GOOS` configured on the command-line to an environment variable using `os.Setenv`.
So, `if os.Getenv(GOOS) == "windows"` can be used.
Then, it is unnecessary to change the function signature.

Reply:
I was trying to emphasize Windows has a unique effect on the binary path. But yeah, that won't add value here.